HealthSDK/GiniHealthSDK/Package.swift

```diff
@@ -6,7 +6,7 @@ import PackageDescription
 let package = Package(
     name: "GiniHealthSDK",
     defaultLocalization: "en",
-    platforms: [.iOS(.v12)],
+    platforms: [.iOS(.v13)],
```

Comment:
[Clarification]: Was iOS 13 required here?

Reply:
Yes, because `GiniPaymentComponents` has minimum version set to v13